# Laelaps laelaps_control/Path2D
#
# A 2D pose specifies position x,y and orientation theta. A timestamped header
# is also provided. A series of these poses define a path.

Header            header    # standard header
Pose2DStamped     pose      # 2D poses
